fitnessllm_dataplatform/utils/logging_utils.py

```
@@ -41,0 +1,131 @@
+            level: Logging level
+        """
+        self.logger = setup_logger(name, level)
+        self.name = name or "root"
+
+    def _format_log(
+        self,
+        level: str,
+        message: str,
+        data_source: Optional[str] = None,
+        user_id: Optional[str] = None,
+        **kwargs
```

Ruff (0.8.2)

Missing type annotation for `**kwargs`

(ANN003)